api/v1/swagger.yaml

```diff
@@ -1341,23 +1327,20 @@ paths:
           required: true
           schema:
             type: object
-            description: Allowable fields when PATCH collector.
+            description: Fields permitted for patching.
             properties:
               description:
                 type: string
                 maxLength: 4096
-                description: >
-                  Describes this Collector; helps a user determine which Collectors they should assign to run their collection.
+                description: Access restricted to Refocus Collector only.
```

Comment:
No... for POST it should only say "access restricted..." but for PATCH/PUT it should have the actual field description.